components/avatar/avatar.css

```diff
@@ -0,0 +1,61 @@
+
+.img-avatar{
+    margin:2rem 0 2rem 0;
+  }
+  span{
+    color:#ff758f;
+  }
+  
+  /* avatar */
+  .avatar-round-xlg, .avatar-round-lg, .avatar-round-md, .avatar-round-sm, .avatar-round-xsm{
+    border-radius: 50%;
+    /* display: block;
+    margin-left: auto;
+    margin-right: auto; */
+  }
+  .avatar-round-xlg, .avatar-square-xlg {
```

Comment:
To refactor this you can have .avatar-lg with height and width and then use it with avatar-round or avatar-square.

Reply:
Agree! Change in approach is required. A lot of classes are becoming redundant here because of this issue. Try to re-use classes as much as possible and give generic names to the classes that are to be re-used elsewhere like Humza suggested

Reply:
Thanks Hamza and Rohit for this review. Changed it like you said.